app.go

```diff
@@ -115,10 +128,7 @@ func (app *App) AddHandler(spec Spec) error {
 		route.Headers(spec.Headers...)
 	}
 
-	// vulcan registration
-	if app.registry != nil && spec.Register != false {
-		app.registerLocation(spec.Methods, spec.Path, spec.Scopes, spec.Middlewares)
-	}
+	app.registerLocation(spec.Methods, spec.Path, spec.Scopes, spec.Middlewares)
```

Comment:
Does that mean that we always register now? Sometimes I need to turn it off (e.g. when using scoll in Vulcand itself)